test/testcafe/framework/page-objects/YubiKeyAuthenticatorPageObject.js

```diff
@@ -9,7 +10,22 @@ export default class YubiKeyAuthenticatorPageObject extends BasePageObject {
     return this.form.setTextBoxValue(name, value);
   }
 
-  submit() {
-    return this.form.clickSaveButton();
+  async waitForYubikeyView() {
+    await this.form.el.find('form').exists;
+  }
+
+  clickEnrollButton() {
+    if(userVariables.v3) {
+      return this.form.clickButton('Set up');
+    }
+    return this.clickVerifyButton();
+  }
+
+  clickVerifyButton() {
+    return this.form.clickButton('Verify');
+  }
+
+  errorBoxTextExists() {
+    return this.form.getByTextOnScreen('We found some errors. Please review the form and make corrections.').exists;
```

Comment:
Is there a better -- more semantic -- way to grab the error box? This seems 
1. too closely tied to the translation
2. implicitly testing the message

Reply:
yeah the problem was there are two elements with role="alert", one being the error box and the other being the error element under the input field.  This was causing the findByRole method to not grab the error box correctly.  Ill see if I can find another way.

Reply:
also the error box is not inside the `form`

Reply:
1. Should the error box be inside the form?
2. Isn't this the page object?

I'm ok if this is following an existing pattern, but it might be revealing an a11y or util structure issue?

Reply:
I checked in V2 and confirmed the error box there does reside inside the form element.  I found that in V3 we are using the InfoSection component for displaying client side errors such as empty input cases which this spec is testing for.  This InfoSection component is not currently placed inside the form element.  I think if we move this component to inside the form it should solve this problem.

Reply:
then we should be able to use the original assertion.

Reply:
Moved the `InfoSection` component here so client side errors will be inside the form element and match V2.